app/views/schedules/index.html.erb

```diff
@@ -1,9 +1,20 @@
 <h1>Watering Schedule</h1>
+<div class="row flex-wrap sticky-top garden-selector">
+  <h3>Gardens to Display </h3>
+  <% current_user.gardens.each do |garden| %>
+    <div id="garden-<%= garden.id %>" class="select-garden">
+      <%= garden.name %>
+    </div>
+  <% end %>
+</div>
 <div class="watering-container">
   <% @days.each do |day| %>
     <div class="<%= day.css_classes %>" name='<%= day.css_name %>'  id='<%= day.css_id %>'>
-      <h3><%= day.day_of_week_name %></h3>
-      <h6><%= day.small_date %></h6>
+      <h3 class="weekday"><%= day.day_of_week_name %></h3>
+      <h6 class="date"><%= day.small_date %></h6>
+      <% if day.css_classes.include?('collapsible') %>
+        <i class="fas fa-caret-down"></i>
```

Comment:
🥕

Reply:
Hahah I ALMOST actually made it a carrot icon but figured that might be a bit obscure